test/e2e/servicebindingrequest_test.go

```diff
@@ -450,7 +461,7 @@ func inspectSBRStatus(
 ) {
 	err := retry(10, 5*time.Second, func() error {
 		t.Logf("Inspecting SBR: '%s'", namespacedName)
-		err := assertSBRStatus(ctx, f, namespacedName)
+		err := assertSBRStatus(ctx, t, f, namespacedName)
```

Comment:
Test functions should always receive `t` as first argument.

```suggestion
		err := assertSBRStatus(t, ctx, f, namespacedName)
```

Reply:
Please disregard this comment; `ctx` should always be the first argument per https://github.com/golang/go/wiki/CodeReviewComments#contexts.